lib/msal-react-native-android-poc/README.md

```diff
@@ -0,0 +1,98 @@
+We can access the username of the currently signed in user and log it to console (for example):
+```javascript
+try {
+    var account = await MSAL.signIn(scopesValue);
+    console.log("Username: " + account.username);
+} catch (error) {
+    console.log(error);
+}
+```
+Since a user is now signed in, we can try to acquire an access token silently and make a call to Microsoft Graph API:
+```javascript
+try {
+    var result = await MSAL.acquireTokenSilent(scopesValue);
+    //api call to MSGraph
+    var response = await fetch (
+        "https://graph.microsoft.com/v1.0/me", {
+             method: 'GET',
+             headers: {
+                Authorization: `Bearer ${result.accessToken}`
+             }
+        }
+    );
+    
+    ... //do something with response here
+    
+} catch (error) {
+  console.log(error);
+}
+```
+Please see msal-react-native-android-poc-sample in the sample folder for an example app that demonstrates the MSAL methods.
+
+For a complete listing of each method's return value and map keys, see the Methods(link to Methods) section.
+
+## What to expect from this library
+Because this library is a proof of concept, it is scoped to specific features and does not implement all the capabilities of an official MSAL for React Native library. 
+
+This proof of concept is scoped to consider:
+* Compatibility with the Android platform
+* Authenticating users with work and school Azure AD accounts (AAD)
+* Single-account mode
+
+This proof of concept does not implement:
+* Compatibility with iOS
+    * To allow compatibility with iOS, an [Objective-C native module](https://reactnative.dev/docs/native-modules-ios) must be created that wraps the       [MSAL for iOS and macOS](https://github.com/AzureAD/microsoft-authentication-library-for-objc#:~:text=The%20MSAL%20library%20for%20iOS%20and%20macOS%20gives,for%20those%20using%20our%20hosted%20identity%20management%20service.).
+* B2C or AD FS
+    * For implementing B2C, refer to the Microsoft docs on using [MSAL for Android with B2C](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-android-b2c). Since the returned value from acquiring tokens is an [`IAuthenticationResult`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.identity.client.authenticationresult?view=azure-dotnet), the results of the wrapped B2C methods can be handled in the same manner as the existing AAD methods.
+* Multiple-account mode
+    * This could be as simple as adding an instance of [MultipleAccountPublicClientApplication](https://docs.microsoft.com/en-us/java/api/com.microsoft.identity.client.multipleaccountpublicclientapplication?view=azure-java-stable) to the existing native module file and      wrapping its corresponding methods.
+    
+Additionally, with this library being a proof of concept, it is not recommended to be used for React Native applications intended for production.
```

Comment:
I think this is an important point that could be bolded and moved to the top of this Readme

Reply:
+1

Reply:
+1

Reply:
Done!